Coloma/Properties/AssemblyInfo.cs

```diff
@@ -32,4 +32,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.1")]
+[assembly: AssemblyVersion("1.0.0.3")]
```

Comment:
There is another PR that increments to this 2, which is why I moved this to 3. In retrospect, however, there's no breaking changes here, and no reason to increment this version at all.